cpp/CMakePresets.json

```diff
@@ -122,6 +122,14 @@
         "ARROW_FLIGHT_SQL": "ON"
       }
     },
+    {
+      "name": "features-flight-sql-odbc",
+      "inherits": "features-flight-sql",
+      "hidden": true,
+      "cacheVariables": {
+        "ARROW_FLIGHT_SQL_ODBC": "ON"
+      }
+    },
```

Comment:
Could you remove this and add only `"ARROW_FLIGHT_SQL_ODBC": "ON"` to `features-maximal` instead?

Reply:
fixed the code